ext/standard/basic_functions.stub.php

```diff
@@ -66,9 +66,9 @@ function ksort(array &$stack, int $sort_flags = PHP_SORT_REGULAR): bool {}
 
 function count(mixed $array, int $mode = COUNT_NORAML): bool {}
 
-function natsort(array &$stack){}
+function natsort(array &$stack): void {}
```

Comment:
Is ir `void` or `bool`?

Reply:
It is `bool`, but actually only TRUE will ever be returned. It might be sensible to change it to `void`, but that would be a BC break.

Reply:
@cmb69 Yeah, this goes to our list at https://bugs.php.net/bug.php?id=75958